blinkit-app/src/DATA/IMAGES.js

```diff
@@ -0,0 +1,30 @@
+export const images = [
```

Comment:
images <> IMAGES

Reply:
We name the files unless its a component or a class in camelCase. The component files are named in PascalCase

Reply:
Fix this whereever applicable in the PR